updates/testing.json

```diff
@@ -19,6 +19,16 @@
           "reason": "https://github.com/coreos/fedora-coreos-streams/issues/30"
         }
       }
+    },
+    {
+      "version": "32.20200601.2.1",
+      "metadata": {
+        "rollout": {
+          "start_epoch": 1591200000,
+          "start_percentage": 0.0,
+          "duration_minutes": 5760
```

Comment:
note the extra long rollout window here

Reply:
That's 4 days, I'm not opposed to this just for safety. Half of it is during the weekend though.

Reply:
could scale back to 3 days if that would be preferable

Reply:
No strong feeling either, though I agree it makes sense to keep it within when we'll actually be online if we're extra worried of regressions. Making it longer means it'll be longer before a node hits an issue.

Reply:
scaled back to 3 days now in the latest upload